atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -151,31 +117,21 @@ const CreateCrashRecord = ({ client }) => {
 
           // Build an array of persons objects formated as a string
           // so the String can be interpolated into the gql tag syntax.
-          let personObjects = "[";
-          let unitObjects = "[";
+          let unitObjects = "";
```

Comment:
if it can be done easily enough, it would be nice to move this payload-building code into a separate function just to keep things readable